crawler/db.py

```diff
@@ -80,24 +89,77 @@ def get_mongo_collection(database: Database, collection_name: str) -> Collection
     return database[collection_name]
 
 
-def copy_collection(database: Database, collection: Collection) -> None:
-    """Copy a collection to a timestamped version of itself.
+def rename_collection_with_suffix(
+    database: Database, collection: Collection, suffix: str = current_time()
+) -> None:
+    """Renames a collection to a timestamped version of itself
 
     Arguments:
-        database {Database} -- the database of the collection to copy
-        collection {Collection} -- the collection to copy
+        database {Database} -- the database of the collection to rename
+        collection {Collection} -- the collection to rename
     """
-    cloned_collection = f"{collection.name}_{datetime.now().strftime('%y%m%d_%H%M')}"
+    new_name = f"{collection.name}_{suffix}"
 
-    logger.debug(f"Copying '{collection.name}' to '{cloned_collection}'")
+    rename_collection(database, collection, new_name)
+    return None
+
+
+def rename_collection(database: Database, collection: Collection, new_name: str) -> None:
+    """Renames a collection to the new name.
```

Comment:
Is database argument needed?

Reply:
Good catch. Nope, doesn't appear to be, think I was using the method signature of the original copy approach.